src/libaktualizr/campaign/CMakeLists.txt

```diff
@@ -0,0 +1,8 @@
+set(SOURCES campaign.cc)
+set(HEADERS campaign.h)
+
+add_library(campaign OBJECT ${SOURCES})
+
+add_aktualizr_test(NAME campaign SOURCES campaign_test.cc ARGS ${CMAKE_CURRENT_SOURCE_DIR}/test)
```

Comment:
test -> tests? But if it works with typo, you don't need it

Reply:
Which one? `${CMAKE_CURRENT_SOURCE_DIR}/test`?

It's to get the path of `https://github.com/advancedtelematic/aktualizr/blob/7acdc4c6c90098e055dad96abed4cf3dc385dbbf/src/libaktualizr/campaign/test/campaigns_sample.json`

Reply:
Sorry I thought it is project source directory